packages/react-components/src/text.ts

```diff
@@ -78,19 +78,18 @@ export const headlineStyles = {
 } as const;
 
 export type AccentColorName =
-  | 'berry'
```

Comment:
Reverted somewhat recent change adding `charcoal` by @fampinheiro - it's not an accent color, it's the default on light theme. I've handled the case it was added for with an IMO better structure